### PR TITLE
Release textarea component

### PR DESCRIPTION
**Pull request by rtkmrussell**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,7 +36,7 @@
     "react-is": "^16.11.0"
   },
   "devDependencies": {
-    "@babel/cli": "7.2.0",
+    "@babel/cli": "7.8.4",
```
Comment (rtkaaho) [inline package.json]: how did this babel change come through ?

Comment (rtkmrussell) [reply package.json]: That's peculiar 🤔   Let me check my package.json and `rm -rf node_modules`

Comment (rtkmrussell) [reply package.json]: I switched to doing `npm ci` and it seems to be okay now...  I should just start using `yarn` 😐

Comment (rtkaaho) [reply package.json]: i havent been using yarn for this. But I soon might change it to use yarn :)

**Pull request merged**